lib/modulesync/repository.rb

```diff
@@ -163,6 +163,17 @@ def submit_changes(files, options)
       true
     end
 
+    def push(branch:, remote_branch:)
+      raise ModuleSync::Error, 'Repository must be locally available before trying to push' unless cloned?
+
+      remote = 'origin'
```

Comment:
I wish we had a constant for this, but I don't expect you to do that in this PR. However, perhaps in this PR it can be, at least in the private method a parameter with a default (so `push(..., remote_name: 'origin')` or similar) so it will be easier in the future?

Reply:
Done.